.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,4 @@
+name: 'GitHub Desktop CodeQL config'
+
+paths-ignore:
+  - '/vendor/'
```

Comment:
Maybe this needs to be something like this?

```suggestion
  - './vendor/'
```

Reply:
I have no clue, it  was copied straight from https://help.github.com/en/github/finding-security-vulnerabilities-and-errors-in-your-code/configuring-code-scanning but https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths seems to suggest it's supposed to be a pattern. Maybe something like this?

```suggestion
  - '/vendor/'
  +'/vendor/yarn*.js'
```

Reply:
Actually, looking at https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet it seems like the patterns are relative to the repository root.

```suggestion
  - '/vendor/'
  - 'vendor/yarn*.js
```